ribbon/frontend/src/main/java/org/wildfly/swarm/examples/servlet/FrontendServlet.java

```diff
@@ -0,0 +1,47 @@
+package org.wildfly.swarm.examples.servlet;
+
+import java.io.InputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.channels.WritableByteChannel;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/*")
+public class FrontendServlet extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String pathInfo = req.getPathInfo();
+        if (pathInfo.equals("/")) {
+            pathInfo = "/index.html";
+        } 
+        URL url = Thread.currentThread().getContextClassLoader().getResource(pathInfo);
+        try (InputStream is = url.openStream()) {                 
+            copy(Channels.newChannel(is), Channels.newChannel(resp.getOutputStream())); 
```

Comment:
I always found writing a servlet to serve static content pretty disappointing, but I guess you tried everything else and it didn't work :-)

Reply:
Yes, the removed Undertow handler could not get the static resources itself when the frontend was launched as a SwarmProcess (as opposed to manually). Some trick might exist, but looking for it can easily burn another week :-). 
In general though I'd have np having a JAX-RS service using a StreamingOutput to return the static resources - esp if the access to these resources requires more than having a default servlet to serve them...

Reply:
I guess I can replace that with the JAX-RS resource after all to avoid doing the manual copying...